src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -65,12 +63,10 @@
                        onClick)
             :onKeyDown (when (and onClick (not disabled?))
                          (common/create-key-handler [:space :enter] onClick))}
-        (or text (icons/icon-text icon))
-        (when (= type :add)
-          [:span {:style {:display "inline-block" :height "1em" :width "1em" :marginLeft "1em"
-                          :position "relative"}}
-           [:span {:style {:position "absolute" :top "-55%" :fontSize "200%" :fontWeight "normal"}}
-            "+"]])]))})
+        text
+        (some->> icon (icons/icon {:style (if text
+                                            {:fontSize 24 :margin "-0.5em -0.3em -0.5em 0.5em"}
+                                            {:fontSize 20})}))]))})
```

Comment:
Changed the button to be able to take text and an icon key.  Got rid of ":type :add" and just made it the plus icon.